### ISSUE TITLE
Hotline

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#16]
## User story
1. As a user that wants to report a problem/incident or wants to create a service request via phone
2. I want / need hotline that I can call and report my problem / explain what I need
3. So that a ticket is created

## Acceptance criteria
- [ ] The "number" hotline is picking up
- [ ] the call will not end prematurely (before the whole problem was stated)
- [ ] A ticket is created with the information given over the call 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
